src/components/SearchSploit/SearchSploit.tsx

```diff
@@ -1,137 +1,117 @@
 import { useForm } from "@mantine/form";
-import { useCallback, useState } from "react";
+import { useCallback, useState, useEffect } from "react";
 import { CommandHelper } from "../../utils/CommandHelper";
 import ConsoleWrapper from "../ConsoleWrapper/ConsoleWrapper";
-import { Text } from "@mantine/core";
-import { List } from "@mantine/core";
-import { Accordion } from "@mantine/core";
-import { UserGuide } from "../UserGuide/UserGuide";
 import { SaveOutputToTextFile_v2 } from "../SaveOutputToFile/SaveOutputToTextFile";
+import { LoadingOverlayAndCancelButtonPkexec } from "../OverlayAndCancelButton/OverlayAndCancelButton";
+import { checkAllCommandsAvailability } from "../../utils/CommandAvailability";
+import InstallationModal from "../InstallationModal/InstallationModal";
 
-const title = "SearchSploit";
-const description_userguide =
-    "SearchSploit is a command-line tool used for searching through Exploit-DB that allows an offline " +
-    "copy of the exploited database to be withheld. This tool is useful for conducting security assessments " +
-    "on segregated networks and provides several search options within its operation.\n\nInformation on the " +
-    "tool can be found at: https://www.exploit-db.com/documentation/Offsec-SearchSploit.pdf\n\n" +
-    "Using SearchSploit:\n" +
-    "Step 1: Enter a Search Term followed by selecting a Search Option.\n" +
-    "       Eg: data, Exact\n\n" +
-    "Step 2: Select an Output type.\n" +
-    "       Eg: json\n\n" +
-    "Step 3: Select a Non-Searching option.\n" +
-    "       Eg: Mirror\n\n" +
-    "Step 4: Enter an Exploit Database ID.\n" +
-    "       Eg: PLACEHOLDER (required when using the 'Path' output or Non-Search options)\n\n" +
-    "Step 5: Click Scan to commence SearchSploit's operation.\n\n" +
-    "Step 6: View the Output block below to view the results of the tools execution.";
-
-interface FormValuesType {
+// Define form values type
+interface FormValues {
     searchTerm: string;
     searchOption: string;
-    out_put: string;
-    non_search: string;
-    ebd_id: string;
+    outputType: string;
+    nonSearch: string;
+    ebdId: string;
 }
 
-const searchOption = ["Case", "Exact", "Strict", "Title"];
-
-const out_put = ["json", "Overflow", "Path", "Verbose", "www"];
-
-const non_search = ["Mirror", "Examine"];
+// Component Constants
+const title = "SearchSploit";
+const description = "SearchSploit is a command-line tool used for searching through Exploit-DB.";
+const steps = `Step 1: Enter a Search Term followed by selecting a Search Option.
+Step 2: Select an Output type.
+Step 3: Select a Non-Searching option.
+Step 4: Enter an Exploit Database ID.
+Step 5: Click Scan to commence SearchSploit's operation.
+Step 6: View the Output block below to view the results of the tool's execution.`;
+const sourceLink = "https://www.exploit-db.com/documentation/Offsec-SearchSploit.pdf";
```

Comment:
I think you used the wrong link for sourceLink, it should be like this:
```suggestion
const sourceLink = "https://www.kali.org/tools/exploitdb/#searchsploit";
```